source/_patterns/02-organisms/content-headers/content-header.mustache

```diff
@@ -5,6 +5,147 @@
     class="content-header {{#image}}content-header--image content-header--header{{/image}} {{^image}}wrapper {{#header.possible}}content-header--header{{/header.possible}}{{/image}} {{#socialMediaSharers}}content-header--has-social-media-sharers{{/socialMediaSharers}} {{#header.hasProfile}}content-header--has-profile{{/header.hasProfile}} clearfix"
```

Comment:
Any reason not to just add the `content-header-grid` class to the `header` element?

Reply:
This is a conversation I will be having with Nathan and how it will integrate with the rest of the Content Headers as this exercise was just to get the new header in to start